website/app/components/Head.tsx

```diff
@@ -1,11 +1,56 @@
 import { Helmet } from 'react-helmet';
 import { DocumentationLoader } from '~/loaders/documentation.server';
 import codeHikeStyles from '@code-hike/mdx/dist/index.css';
+import { useEffect } from 'react';
 
 export const Head = ({ data }: { data: DocumentationLoader }) => {
   const favicon = getFavicon({ data });
+
+  useEffect(() => {
```

Comment:
Why is this in an effect and not just inlined within Helmet?

Reply:
Can helmet append things to the `body`?